src/tools/kata-ctl/src/ops/volume_ops.rs

```diff
@@ -0,0 +1,78 @@
+use crate::args::{DirectVolSubcommand, DirectVolumeCommand};
+
+use anyhow::{anyhow, Ok, Result};
+use futures::executor;
+use hyper;
+use runtimes;
+use std::time::Duration;
+use url;
+
+const TIMEOUT: Duration = Duration::new(5, 0);
+const CONTENT_TYPE_JSON: &str = "application/json";
+
+pub fn handle_direct_volume(vol_cmd: DirectVolumeCommand) -> Result<()> {
+    let command = vol_cmd.directvol_cmd;
+    match command {
+        DirectVolSubcommand::Add(args) => {
+            runtimes::direct_volume::add(&args.volume_path, &args.mount_info)?;
+        }
+        DirectVolSubcommand::Remove(args) => {
+            runtimes::direct_volume::remove(&args.volume_path)?;
```

Comment:
`Remove` need to delete the direct volume path including all the files inside it.